econml/utilities.py

```diff
@@ -570,7 +573,8 @@ def get_input_columns(X):
     }
     if type(X) in type_to_func:
         return type_to_func[type(X)](X)
-    return None
+    len_X = 1 if np.ndim(X) == 1 else np.asarray(X).shape[1]
+    return ["{}{}".format(prefix, i) for i in range(len_X)]
```

Comment:
I think a format string literal is clearer:
```suggestion
    return [f"{prefix}{i}" for i in range(len_X)]
```